### PR TITLE
fixed flaky test for tags

### PR DESCRIPTION
**Pull request by beblicarl**
#### Pre-Submission Checklist

-   [x] Your pull request targets the `main` branch.
-   [x] Branch name is descriptive and follows the guidelines [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices).
-   [x] All tests pass . Use `git commit --amend` to amend any fixes.

#### Checklist:

-   [x] All tests are passing.
-   [x] Tested changes locally.
-   [x] Improved documentation or readme where necessary.
-   [x] Your commit messages format follows our commit message guideline [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices#commits).

#### What does this PR do?

The success block was declared within the failure block. This PR fixes it so that the two blocks are declared separately
#### How should this be manually tested?

1. Start the test with `npm run test`
2. This successfully returns all test passing 

#### What are the relevant issues?
The success block was declared within the failure block. This PR fixes it so that the two blocks are declared separately

### CODE REVIEW

File: src/routes/tags/fetch-tags.e2e.test.js [hunk 1]
```diff
@@ -2,48 +2,60 @@ const { expect } = require('chai')
...
 	beforeEach(async ()=> {
+
 		user = await fixtures.insertUser()
 		const body = { id: user.id, email: user.email }
 		accessToken = fixtures.generateAccessToken(
 			{user : body}
 		)
 	})
+
 	describe('Failure', ()=> {
```
Comment (jakazzy) [inline src/routes/tags/fetch-tags.e2e.test.js:18]: Add a logger here. Similar to what is here:
https://github.com/tbf-challenge/teamwork-be/pull/300/files#diff-9d469a35ac5ab7213420d0e84ab4e3cf6d1105a603dd131a54bd85e0b0b89b53R85
 To investigate flaky test.

**Comment by jakazzy** [comment]
Chairman, how does the current change fix the flaky test? @beblicarl

File: src/routes/tags/fetch-tags.e2e.test.js (hunk 1 above)
Comment (beblicarl) [reply src/routes/tags/fetch-tags.e2e.test.js:18]: sure

Comment (beblicarl) [reply src/routes/tags/fetch-tags.e2e.test.js:22]: @jocrah , i have implemented suggested changes now, kindly review

**Review by jakazzy — APPROVED** [review]

**Pull request merged**